network-api/networkapi/wagtailpages/templates/wagtailpages/blog_index_page.html

```diff
@@ -1,6 +1,6 @@
 {% extends "./index_page.html" %}
 
-{% load wagtailcore_tags wagtailimages_tags i18n %}
+{% load wagtailcore_tags wagtailimages_tags i18n localization homepage_tags blog_tags static %}
```

Comment:
**Question** Is this necessary anymore? Feels like it was leftover from refactoring.

Reply:
Updated!